cluster/gce/util.sh

```diff
@@ -546,6 +551,29 @@ function get-template-name-from-version() {
   echo "${NODE_INSTANCE_PREFIX}-template-${1}" | cut -c 1-63 | sed 's/[\.\+]/-/g;s/-*$//g'
 }
 
+# validates the NODE_LOCAL_SSDS_EXT variable 
+function validate-node-local-ssds-ext(){
+  ssdopts=$1
+
+  if [[ -z ${ssdopts[0]} || -z ${ssdopts[1]} || -z ${ssdopts[2]} ]]; then
+	  echo -e "${color_red}Local SSD: NODE_LOCAL_SSDS_EXT is malformed, found ${ssdopts[0]-_},${ssdopts[1]-_},${ssdopts[2]-_} ${color_norm}" >&2
+    exit 2
+  fi
+  if [[ ${ssdopts[1]} != "scsi" && ${ssdopts[1]} != "nvme" ]]; then
+    echo -e "${color_red}Local SSD: Interface must be scsi or nvme, found: ${ssdopts[1]} ${color_norm}" >&2
+    exit 2
+  fi
+  if [[ ${ssdopts[2]} != "fs" && ${ssdopts[2]} != "block" ]]; then
+    echo -e "${color_red}Local SSD: Filesystem type must be fs or block, found: ${ssdopts[2]} ${color_norm}"  >&2
+    exit 2
+  fi
+  local_ssd_ext_count=$((local_ssd_ext_count+ssdopts[0]))
+  if [[ ${local_ssd_ext_count} -gt 8 || ${local_ssd_ext_count} -lt 1 ]]; then
```

Comment:
Where is 8 coming from? Can it be a bash constant at the top of the file? How important is this check? Will it require us to change this if GCE supports more?

Reply:
It's just because the max supported local SSDs on GCE is 8. It would require a change if GCE supports more. However, if we remove this check and a user requests more than 8 local SSD's the cluster will fail to initialize without any specific error message.

I will make it a constant.